app/App.hx

```diff
@@ -113,8 +117,13 @@ class App
         #end
         while (true)
         {
+            if(bot.resetFlag==true){
+                bot = new Bot(data.email + ":" + data.key,data.ip + ":" + data.port,data.legacy,relay,seed);
```

Comment:
¹the bot system should really be more robust, and have shared code for combo vs non